src/main.zig

```diff
@@ -1254,6 +1255,14 @@ fn buildOutputType(
                     .framework => try frameworks.append(it.only_arg),
                     .nostdlibinc => want_native_include_dirs = false,
                     .strip => strip = true,
+                    .exec_model => {
+                        wasi_exec_model = if (std.mem.eql(u8, it.only_arg, "reactor"))
+                            Compilation.WasiExecModel.reactor
+                        else if (std.mem.eql(u8, it.only_arg, "command"))
+                            Compilation.WasiExecModel.command
+                        else
+                            null;
```

Comment:
Minor nit: you actually don't need to initialize to `null` here since it is already initialized to this value by default.

Reply:
fixed in  https://github.com/ziglang/zig/pull/8978/commits/230878352006e466f66600a979bdba8bddc98acd